pkg/demo/quill/Library/QuillFontPicker.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC.
```

Comment:
Just a reminder to make this 2022 for new code. Also I think we have a new header? The one below references Polymer. For future edits.